MapboxDirections/MBRouteOptions.swift

```diff
@@ -304,6 +312,13 @@ open class RouteOptions: NSObject, NSSecureCoding {
     open var includesExitRoundaboutManeuver = false
     
     /**
+     An Array of `RoadClass` that the routing engine will attempt to avoid.
+     
+     The order of this Array does not impact the results.
+     */
+    open var excludeRoadClasses: RoadClasses? = []
```

Comment:
“Exclude road classes” sounds like a verb, which makes this Objective-C syntax look like it’s performing an action rather than getting a property value:

```objc
[routeOptions excludeRoadClasses];
```

`excludesRoadClasses` is also problematic, because it sounds like a Boolean (as in, “does this property exclude road classes?”). So `excludedRoadClasses` would be the best name for this property.